nautobot/dcim/form_mixins.py

```diff
@@ -25,7 +25,7 @@ class LocatableModelFormMixin(forms.Form):
     location = DynamicModelChoiceField(
         queryset=Location.objects.all(),
         required=False,
-        # TODO: query_params={"base_site": "$site"},
+        # 2.0 TODO: query_params={"base_site": "$site"},
```

Comment:
Doesn't need to be a 2.0 item, it wouldn't be a breaking change, just an enhancement. Tracked in #2518. Same comment on the next three `2.0 TODO` changes.